client/src/components/ChatMessage.tsx

```diff
@@ -2,21 +2,60 @@ import React from "react";
 
 interface Props {
   text: string;
+  userIndex: number;
 }
 
-const messageStyle = {
-  display: "flex",
-  flexDirection: "column" as "column",
-  width: "200px",
-  alignItems: "flex-end",
-  padding: "10px",
-  borderRadius: "10px",
-  backgroundColor: "#09f",
-  margin: "10px"
+const getMessageStyle = (userIndex: number) => {
+  if (userIndex === 0) {
+    return { backgroundColor: "#ededed", color: "black" };
+  }
+  if (userIndex === 1) {
+    return { backgroundColor: "#09f", color: "white" };
+  }
+  return { backgroundColor: "white", color: "black" };
 };
 
-const ChatMessage: React.FunctionComponent<Props> = ({ text }) => {
-  return <div style={messageStyle}>{text}</div>;
+// const messageStyle = {
```

Comment:
This is a pet peeve but I try to avoid committing too much commented out code

Reply:
You're right. It's unkempt. I'll avoid doing that next time